{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -35,35 +35,42 @@ def get_tm_session(session_factory, transaction_manager, request=None):
     - When using scripts you should wrap the session in a manager yourself.
       For example::
 
-          import transaction
+    .. code-block:: python
 
-          engine = get_engine(settings)
-          session_factory = get_session_factory(engine)
-          with transaction.manager:
-              dbsession = get_tm_session(session_factory, transaction.manager)
+        import transaction
+
+        engine = get_engine(settings)
+        session_factory = get_session_factory(engine)
+        with transaction.manager:
+            dbsession = get_tm_session(session_factory, transaction.manager)
 
     This function may be invoked with a ``request`` kwarg, such as when invoked
```

Comment:
Are this and the subsequent paragraphs supposed to be indented under the list item "When using scripts you should wrap the session in a manager yourself.", or should it be dedented to be part of the function? Indentation puts them under the list item context, and I don't think that is what you intended.